features/manifest.feature

```diff
@@ -0,0 +1,38 @@
+Feature: With a manifest provided
+  Scenario: containing one problem
+    Given a file named "fail.pp" with:
+    """
+    # foo
+    class test::foo { }
+    """
+    When I run `puppet-lint fail.pp`
+    Then it has 1 error and 0 warnings
+
+    Scenario: containing a control statement
+      Given a file named "ignore.pp" with:
```

Comment:
I first did have a version which loaded fixtures, but decided to inline things here for now.

Reply:
I think it is more readable to have the content here :100: .